x-pack/plugin/autoscaling/src/main/java/org/elasticsearch/xpack/autoscaling/storage/ReactiveStorageDeciderService.java

```diff
@@ -714,6 +775,10 @@ public void writeTo(StreamOutput out) throws IOException {
             out.writeString(reason);
             out.writeLong(unassigned);
             out.writeLong(assigned);
+            if (out.getVersion().onOrAfter(Version.V_8_3_0)) {
```

Comment:
nit: can you extract that version to a constant?

Reply:
Done!